ci/python_310_dev.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
     - codecov
     - folium
     - jupyterlab
+    - jupytext=1.13.8
```

Comment:
@christophfink : FYI it's good to hardcode this to specific version of jupytext, because the Markdown files will contain metadata about the Jupytext version. If people have different jupytext versions installed on their computer, there will be changes coming to these files even nothing changes for real. Hence, it's good to ensure everyone working on the docs have identical version of Jupytext installed.